api/test/countries.e2e-spec.ts

```diff
@@ -53,13 +53,13 @@ describe('CountriesModule (e2e)', () => {
     it('Should list administrative areas within a given country', async () => {
       const response = await request(app.getHttpServer())
         .get(
-          `/api/v1/countries/${aCountry.gid0}/administrative-areas?page[size]=25`,
+          `/api/v1/countries/${aCountry.attributes.gid0}/administrative-areas?page[size]=25`,
         )
         .set('Authorization', `Bearer ${jwtToken}`)
         .expect(200);
 
       const resources = response.body.data;
-      expect(resources[0].type).toBe('administrative-areas');
```

Comment:
is this intentional?

Reply:
yes and no - it was ok when I created this PR, but I have eventually amended it to read `admin_areas` in the latest commits before merging #76 as it was by then out of sync with what the API would return